# Translations template for designate.
# Copyright (C) 2015 ORGANIZATION
# This file is distributed under the same license as the designate project.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2015.
#
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: designate 1.0.0.0b2.dev172\n"
"Report-Msgid-Bugs-To: EMAIL@ADDRESS\n"
"POT-Creation-Date: 2015-07-27 06:06+0000\n"
"PO-Revision-Date: YEAR-MO-DA HO:MI+ZONE\n"
"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
"Language-Team: LANGUAGE <LL@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=utf-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Generated-By: Babel 1.3\n"

#: designate/coordination.py:108
msgid "Error sending a heartbeat to coordination backend."
msgstr ""

#: designate/dnsutils.py:106
#, python-format
msgid "Unknown TSIG key from %(host)s:%(port)d"
msgstr ""

#: designate/dnsutils.py:113
#, python-format
msgid "Invalid TSIG signature from %(host)s:%(port)d"
msgstr ""

#: designate/dnsutils.py:120
#, python-format
msgid "Failed to deserialize packet from %(host)s:%(port)d"
msgstr ""

#: designate/dnsutils.py:127
#, python-format
msgid "Unknown exception deserializing packet from %(host)s %(port)d"
msgstr ""

#: designate/dnsutils.py:328
#, python-format
msgid "AXFR timed out for %(name)s from %(host)s"
msgstr ""

#: designate/dnsutils.py:332
#, python-format
msgid "Domain %(name)s is not present on %(host)s.Trying next server."
msgstr ""

#: designate/dnsutils.py:336
#, python-format
msgid "Connection error when doing AXFR for %(name)s from %(host)s"
msgstr ""

#: designate/dnsutils.py:340
#, python-format
msgid "Problem doing AXFR %(name)s from %(host)s. Trying next server."
msgstr ""

#: designate/dnsutils.py:347
#, python-format
msgid "XFR failed for %(name)s. No servers in %(servers)s was reached."
msgstr ""

#: designate/service.py:324
#, python-format
msgid "Unknown exception handling TCP request from: %(host)s:%(port)d"
msgstr ""

#: designate/service.py:354
#, python-format
msgid "Unknown exception handling UDP request from: %(host)s:%(port)d"
msgstr ""

#: designate/service.py:387
#, python-format
msgid "Unhandled exception while processing request from %(host)s:%(port)d"
msgstr ""

#: designate/api/middleware.py:294 designate/api/middleware.py:345
msgid "Missing context in request, please check."
msgstr ""

#: designate/backend/impl_powerdns/migrate_repo/versions/011_records_drop_designate_ids.py:40
msgid "It is not possible to downgrade from schema #11"
msgstr ""

#: designate/manage/tlds.py:138
#, python-format
msgid "Number of errors: %d"
msgstr ""

#: designate/manage/tlds.py:141
#, python-format
msgid ""
"Error Lines:\n"
"%s"
msgstr ""

#: designate/network_api/neutron.py:126
#, python-format
msgid "Failed calling Neutron %(region)s - %(endpoint)s"
msgstr ""

#: designate/pool_manager/service.py:200
msgid "An unhandled exception in periodic recovery occurred"
msgstr ""

#: designate/pool_manager/service.py:236
msgid "An unhandled exception in periodic synchronization occurred."
msgstr ""

#: designate/pool_manager/service.py:298
#, python-format
msgid "Failed to create domain %(domain)s on target %(target)s"
msgstr ""

#: designate/pool_manager/service.py:364
#, python-format
msgid "Failed to update domain %(domain)s on target %(target)s"
msgstr ""

#: designate/pool_manager/service.py:440
#, python-format
msgid "Failed to delete domain %(domain)s on target %(target)s"
msgstr ""

